contentcuration/contentcuration/frontend/channelEdit/components/edit/ResourcesNeededOptions.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+
+  <VSelect
+    ref="need"
+    v-model="need"
+    :items="resources"
+    box
+    chips
+    :label="$tr('resourcesNeededLabel')"
+    multiple
+    deletableChips
+    clearable
+  />
+
+</template>
+
+<script>
+
+  import { ResourcesNeededTypes } from 'shared/constants';
+  import { constantsTranslationMixin, metadataTranslationMixin } from 'shared/mixins';
+
+  //the variable below can be changed or removed when metadata/Kolibri is updated
+  const keysToBeTemporarilyRemoved = ['PEERS', 'TEACHER', 'PRIOR_KNOWLEDGE', 'MATERIALS'];
+  const dropdown = updateResourcesDropdown(keysToBeTemporarilyRemoved) || ResourcesNeededTypes;
```

Comment:
I know it's technically allowed, but could you move the function definition above the invocation here?